Firestore/core/src/firebase/firestore/remote/grpc_metadata_provider.mm

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
This should be `firebase_client_grpc_metadata_provider_apple.mm` to match `FirebaseClientGrpcMetadataProvider` on Android. The `_apple` in the filename allows us to have alternative implementations on non-apple platforms.